### PR TITLE
Update ios-ipados-supervised-device-security-configurations.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the Level 2 verbiage

### CODE REVIEW

**Comment by PRMerger13** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2513

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

File: memdocs/intune/enrollment/ios-ipados-supervised-device-security-configurations.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ To simplify the table below, only configured settings are listed. Undocumented d
 
 ## Supervised enhanced security (Level 2)
 
-Level 2 is the recommended configuration for personal devices where users access more sensitive information. These devices are a natural target in enterprises today. These settings don't assume a large staff of highly skilled security personnel. Therefore, they should be accessible to most enterprise organizations. This configuration is applicable to most mobile users accessing work or school data on a device.
+Level 2 is the recommended configuration for supervised  devices where users access more sensitive information. These devices are a natural target in enterprises today. These settings don't assume a large staff of highly skilled security personnel. Therefore, they should be accessible to most enterprise organizations. This configuration is applicable to most mobile users accessing work or school data on a device.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/ios-ipados-supervised-device-security-configurations.md:85]:
```suggestion
Level 2 is the recommended configuration for supervised devices where users access more sensitive information. These devices are a natural target in enterprises today. These settings don't assume a large staff of highly skilled security personnel. Therefore, they should be accessible to most enterprise organizations. This configuration is applicable to most mobile users accessing work or school data on a device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by lenewsad** [comment]
#sign-off

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**